src/libraries/System.Runtime.Serialization.Xml/tests/XmlDictionaryReaderTests.cs

```diff
@@ -195,6 +196,10 @@ public static void BinaryXml_ReadPrimitiveTypes()
             AssertReadContentFromBinary(new TimeSpan(0x0807060504030201), XmlBinaryNodeType.TimeSpanText, new byte[] { 01, 02, 03, 04, 05, 06, 07, 08 });
             AssertReadContentFromBinary(new decimal(0x20212223, 0x10111213, 0x01020304, true, scale: 0x1b), XmlBinaryNodeType.DecimalText
                 , new byte[] { 0x0, 0x0, 0x1b, 0x80, 0x4, 0x3, 0x2, 0x1, 0x23, 0x22, 0x21, 0x20, 0x13, 0x12, 0x11, 0x10 });
+            DateTime datetime = new DateTime(2022, 8, 26, 12, 34, 56, DateTimeKind.Utc);
+            Span<byte> datetimeBytes = stackalloc byte[8];
+            BinaryPrimitives.WriteInt64LittleEndian(datetimeBytes, datetime.ToBinary());
+            AssertReadContentFromBinary(datetime, XmlBinaryNodeType.DateTimeText, datetimeBytes);
```

Comment:
To properly test this, you really should test using an inline byte array like the other tests. All you are testing here is that WriteInt64LittleEndian and ReadInt64LittleEndian round trip data.